data/core/encyclopedia/geography.cfg

```diff
@@ -117,7 +149,9 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
 [topic]
     id=northlands
     title= _ "Northlands"
-    text= _ "There is no government of the Northlands. Various groups of orcs, dwarves, barbarian men and even elves populate the region. The northern and eastern borders are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>.
+    text= _ "The Northlands are filled with unstable clans in endless conflict. It is an unlawful and perilous place plagued by its cold, icy terrain and constant war. Various groups of orcs, dwarves, men, gryphons, ogres, and even elves fight over the region. The northern and eastern borders of the Northlands are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>.
+
+    Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish Kingdoms of Knalga and Kal Kartha are exceptions to their chaotic surroudings, providing both heat and protection to their people. While they may often be in conflict with orcs and trolls, the heart of Dwarvish territory is considered relatively safe in comparison.
 
 <header>text='Notable cities:'</header>
```

Comment:
As the string is already changing, please split the Notable Cities and Notable Land Features sections into separate strings.